meson_options.txt

```diff
@@ -198,13 +198,13 @@ option(
 option(
     'with-dbus-daemon-path',
     type: 'string',
-    value: '/bin/dbus-daemon',
+    value: '',
     description: 'Set path to D-Bus daemon',
 )
 option(
     'with-dbus-sysconf-path',
     type: 'string',
-    value: '/etc/dbus-1/system.d',
+    value: '',
```

Comment:
It's a bit unfortunate that the default value isn't defined here anymore, since we lose the transparency to users. I assume you did this because you want a fallback for macOS?

Reply:
The value: argument in meson_options.txt may only be a string, int, bool, or array of those. The default dbus sysconf path is a concatenation of the `prefix` variable and `'etc/dbus-1/system.d'` so we have to leave it as an empty string. We can keep the default bus-daemon path though as it is just a pure string.